core-bundle/src/Event/PreviewUrlConvertEvent.php

```diff
@@ -21,6 +22,13 @@ class PreviewUrlConvertEvent extends Event
      */
     private $url;
 
+    private $request;
+
+    public function __construct(Request $request)
```

Comment:
What is the advantage of storing the request in the event instead of injecting the request stack? Convenience?

Reply:
It's just an integral part of that event. Just like it is for `kernel.request`, `kernel.response` etc.